lib/errors.js

```diff
@@ -73,6 +73,8 @@ class HTTPError extends Error {
 
     res.write(content);
     res.end();
+
+    Log.debug(err);
```

Comment:
Yes. `Error.handler` is only called internally by the controller: https://github.com/rapid7/turnstile/blob/master/lib/control/layer.js#L65-L70